addons/im_livechat/models/discuss_channel.py

```diff
@@ -530,6 +530,36 @@ def _chatbot_validate_email(self, email_address, chatbot_script):
             'error_message': error_message,
         }
 
+    def _add_members(
+        self,
+        *,
+        guests=None,
+        partners=None,
+        users=None,
+        create_member_params=None,
+        invite_to_rtc_call=False,
+        post_joined_message=True,
+        inviting_partner=None,
+    ):
+        all_members = super()._add_members(
+            guests=guests,
+            partners=partners,
+            users=users,
+            create_member_params=create_member_params,
+            invite_to_rtc_call=invite_to_rtc_call,
+            post_joined_message=post_joined_message,
+            inviting_partner=inviting_partner,
+        )
+        for channel in self:
+            # sudo: discuss.channel - accessing livechat_status in internal code is acceptable
+            if (
+                not channel.livechat_end_dt
+                and channel.sudo().livechat_status == "need_help"
```

Comment:
Good thought to take the closed conversation in mind. However `livechat_status` is necessarily empty when `livechat_end_dt` it set (`_livechat_end_dt_status_constraint`) so the extra condition is not necessary.